### ISSUE TITLE
Mocks for "Application Submitted" page

### ISSUE THREAD

**Issue by ssandbekkhaug** [civiform/civiform#7735]
Create North Star mocks for the "Application Submitted" page

# Old UX

<img width="1360" alt="Application_Submitted" src="https://github.com/civiform/civiform/assets/23390490/be82ee78-b1ef-47cb-b0fd-968f23d6f01b">

**Comment by ssandbekkhaug** [comment]
ETA Monday, June 17

**Comment by sijiayam** [comment]
@ssandbekkhaug I believe we have some use cases where a program submission confirmation page has suggested relevant programs. I assume we would need a design for those?

**Comment by sijiayam** [comment]
I have two mocks available but need to confirm with product that all feature are captured accurately: 

Features include:

- Ability for applicants to download/print confirmation of application
- Ability for applicants to create an account/login to save application to an account
- Ability for applicants to view and apply for relevant programs
- Ability for admins to customize text on what happens next(?)

![Image](https://github.com/civiform/civiform/assets/116591382/7ce6c936-0b2d-408d-ba7f-aa3c63c1da47)

@shreyachatterjee00 @isabelcshaw Can either of you: 
- Confirm/approve that this mock replaces existing functionality appropriately?
- Proposed UX of new layout supports improving applicant experience

**Comment by ssandbekkhaug** [comment]
> @ssandbekkhaug I believe we have some use cases where a program submission confirmation page has suggested relevant programs. I assume we would need a design for those?

Yes, I think only the "common intake" page shows suggested programs at the moment: https://github.com/civiform/civiform/issues/7736

Do you think the "regular" application submitted page should also show "More programs to apply to"? If so, can that be a post-launch feature?

**Comment by sijiayam** [comment]
@isabelcshaw We need product approval on the new mocks to unblock development
